fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -767,6 +767,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         true,
     )
 
+    val remoteSettingsServer by booleanPreference(
```

Comment:
let's update the name of this to reflect it's a boolean
```suggestion
    val useProductionRemoteSettingsServer by booleanPreference(
```